creator-node/prod.env

```diff
@@ -28,3 +28,4 @@ delegateOwnerWallet=
 delegatePrivateKey=
 discoveryProviderWhitelist=
 spOwnerWallet=
+enableRsyslog=0
```

Comment:
can we use true / false instead?

Reply:
Hey! I thought about that and we're having that conversation here:

https://github.com/AudiusProject/audius-protocol/pull/3124#discussion_r882909502

Reply:
Also, required values are at the bottom and left unset, pls move this to the above section, thanks